mewarpx/medocker/docker_shared/run_warpx.sh

```diff
@@ -119,11 +130,15 @@ do
     find . -type f -exec touch -a -c {} +
 done
 }
-# https://stackoverflow.com/questions/21465297/tee-stdout-and-stderr-to-separate-files-while-retaining-them-on-their-respective
```

Comment:
I think it would still be useful to keep this link for future reference.

Reply:
Sounds good, I'll put it back. I took it out because I initially thought the changes below it made the comment invalid but that is not the case.